psi4/src/psi4/libfock/MemDFJK.cc

```diff
@@ -53,8 +53,9 @@ using namespace psi;
 
 namespace psi {
 
-MemDFJK::MemDFJK(std::shared_ptr<BasisSet> primary, std::shared_ptr<BasisSet> auxiliary)
-    : JK(primary), auxiliary_(auxiliary) {
+MemDFJK::MemDFJK(std::shared_ptr<BasisSet> primary, std::shared_ptr<BasisSet> auxiliary,
+    Options& options) : JK(primary), auxiliary_(auxiliary), options_(options) {
```

Comment:
Options remains in the MemDFJK constructor?

Reply:
This actually opens up a potential route of discussion! 

Yes, options remains in the MemDFJK constructor. Of the JK derived classes, all of them contain an options_ member variable, _except_ for GTFockJK, MemDFJK, and DiskDFJK (and CDJK by extension). I originally added options to the MemDFJK constructor to define its own options_ member variable, which was then passed to DFHelper for this PR. However, even with the removal of explicit options-passing to DFHelper, having an options_ member variable for MemDFJK is still reasonable from a standardization perspective. in my opinion.

What are your thoughts on the matter?

Reply:
ah, thanks for the clarification! yes, I agree with adding `options_` for standardization. it'll be easier to deal with these classes if they have consistent member data.

Reply:
Yes, I would agree! In that case, I will keep this as it is, then.